tests/test_queries/test_queryset_level_methods.py

```diff
@@ -188,6 +189,28 @@ async def test_get_or_create():
             )
 
 
+@pytest.mark.asyncio
+async def test_get_or_create_with_defaults():
```

Comment:
A missing path is same arguments in kwargs and in _defaults

Reply:
Sorry I couldn't understand what do you mean

Reply:
right now you miss a test for `get_or_create(_defaults={"name":"test2"}, name="test")`.
Same argument in kwargs and defaults -> it will raise exception argument name provided multiple times to method create in you current implementation.

Reply:
I've added a test for overwriting `kwargs` by `_defaults`.